src/index.ts

```diff
@@ -731,8 +749,7 @@ export class ValidationService {
 
             //Prevent the submit before validation
```

Comment:
This comment is no longer correct.
```suggestion
            //Calls `preValidate` before validation. This provides a means to prevent submit before validation.
```